glide.lock

```diff
@@ -10,6 +10,8 @@ imports:
   version: 4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9
   subpackages:
   - quantile
+- name: github.com/ccbrown/go-immutable
+  version: e9015daa17c46c136f648b80536c5f1814002fea
```

Comment:
I have a nice immutable, ordered map that works really well for my in-memory store implementation.